### PR TITLE
Add Deoxys and Castform's weather forms.

### PR DESCRIPTION
**Pull request by 123FLO321**
## Description
This PR displays the correct form for Deoxys and Castform on the frontend just like it's done for unknowns

## How Has This Been Tested?
Did not test this yet but It's pretty straightforward.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: static/js/map.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ const audio = new Audio('static/sounds/ding.mp3')
 const cryFileTypes = ['wav', 'mp3']
 
 const genderType = ['♂', '♀', '⚲']
-const unownForm = ['unset', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', '!', '?']
+const forms = ['unset', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', '!', '?', i8ln('Normal'), i8ln('Sunny'), i8ln('Rainy'), i8ln('Snowy')]
```
Comment (billyjbryant) [inline static/js/map.js]: Might as well add Deoxys forms to this as well to prepare for the future. 

DEOXYS_NORMAL = 33
DEOXYS_ATTACK = 34
DEOXYS_DEFENSE = 35
DEOXYS_SPEED = 36

Comment (pogo-excalibur) [reply static/js/map.js]: To clarify the above (if you're unfamiliar with the format):

Add `i8ln('Normal'), i8ln('Attack'), i8ln('Defense'), i8ln('Speed')` to the end of the forms array.

Comment (123FLO321) [reply static/js/map.js]: done

**Comment by BrownSlaughter** [comment]
just wondering instead of having the word which takes up a bit of space, should we use icons
👤☀️☔️⛄️
Then we have no translation problems

example
image

deoxy could be
👤 ⚔️ 🛡️🗲

**Comment by 123FLO321** [comment]
changed @sebastienvercammen @BrownSlaughter

File: static/js/map.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ const audio = new Audio('static/sounds/ding.mp3')
 const cryFileTypes = ['wav', 'mp3']
 
 const genderType = ['♂', '♀', '⚲']
-const unownForm = ['unset', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', '!', '?']
+const forms = ['unset', 'A', 'B', 'C', 'D', 'E', 'F', 'G', 'H', 'I', 'J', 'K', 'L', 'M', 'N', 'O', 'P', 'Q', 'R', 'S', 'T', 'U', 'V', 'W', 'X', 'Y', 'Z', '!', '?', '👤', '☀️', '☔️', '☃️', '👤', '⚔️', '🛡️', '⚡️']
```
Comment (fosJoddie) [inline static/js/map.js]: use `⛄️` for the snow form for consistency with other weather emoji uses

**Pull request merged**